aws-s3-private-bucket/main.tf

```diff
@@ -1,4 +1,6 @@
 locals {
+  use_grant_acl = length(var.canonical_user_id_and_permissions) == 0 ? false : true
```

Comment:
I would rewrite as:

```suggestion
  acl = length(var.canonical_user_id_and_permissions) == 0 ? null : "private"
```

Reply:
heads up the order was wrong, fixed in the latest commit